Meeting widget Implementation (#13)
webex/widgets: src/widgets/WebexMeeting/README.md

# Webex Meeting Widget

Webex Meeting widget displays the complete, default Webex meeting experience.

<p align="center">
  <img src="./WebexMeeting.gif" alt="Default Webex Meeting" />
</p>

## Preview

> To see all the different possible states of the Webex Meeting component, you can run our start script:

Reviewer: ```suggestion
To get a preview of the Webex Meeting widget, you can run our start script:
```

```shell
  npm start
```

## Embed

> - import the widget from the `@webex/widgets` library and embed it int your React application

Reviewer: ```suggestion
Import the Meeting widget from the `@webex/widgets` library and embed it in your React application
```
